src/Cli/dotnet/Commands/Test/Terminal/AnsiCodes.cs

```diff
@@ -128,10 +128,10 @@ internal static class AnsiCodes
     public const string RemoveBusySpinner = $"{Esc}]9;4;0;{Esc}\\";
 
     public static string Colorize(string? s, TerminalColor color)
-        => String.IsNullOrWhiteSpace(s) ? s ?? string.Empty : $"{CSI}{(int)color}{SetColor}{s}{SetDefaultColor}";
+        => string.IsNullOrWhiteSpace(s) ? s ?? string.Empty : $"{CSI}{(int)color}{SetColor}{s}{SetDefaultColor}";
```

Comment:
Nice catch!

Reply:
This was done automatically from the tooling when it scanned the file.